tests/manage/test_pvc_delete_verify_size_is_returned_to_backendpool.py

```diff
@@ -0,0 +1,179 @@
+"""
+A test case to verify after deleting pvc whether
+size is returned to backend pool
+"""
+import logging
+import time
+import pytest
+
+from ocs import constants, defaults
+import ocs.exceptions as ex
+from tests import helpers
+from ocsci.testlib import tier1, ManageTest
+from utility import templating
+from resources import pod
+from ocs import ocp
+from tests.fixtures import (
+    create_rbd_storageclass, create_ceph_block_pool,
+    create_rbd_secret
+)
+
+logger = logging.getLogger(__name__)
+_templating = templating.Templating()
+
+PV = ocp.OCP(
+    kind='PersistentVolume', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+
+def check_ceph_used_space():
+    """
+    Check for the used space in cluster
+    """
+
+    # ToDo: after running IO's or deleting pod & pvc takes some time
+    time.sleep(20)
```

Comment:
what is happening in this time?

Reply:
@RazTamir I have observed it takes some time for rebalance. When we run IO and check the ceph status immediately it won't be same amount as the data written. Similarly when we delete pod and pvc it takes some time to remove data completely

Reply:
what is it that we are checking in ceph status? some size?
I want to understand if that a specific value we can sample and in case it has no change for let's say 3 samples, we know the rebalance finished
makes sense?

Reply:
yes, In ceph status need to check the used size. Made changes accordingly